Functions.Templates/Templates/TimerTrigger-ExcelOutlook/metadata.json

```diff
@@ -0,0 +1,61 @@
+﻿{
+  "defaultFunctionName": "TimerTrigger-ExcelOutlook",
+  "description": "$Read two Excel tables, send emails based on the data, and then update an Excel table.",
+  "name": "Send Emails",
```

Comment:
The name is very ambiguous and non descriptive, we already have templates that send emails. It would be okay to have a longer name if it needs to be.